### PR TITLE
added login and register pages ui

### PR DESCRIPTION
**Pull request by nippaR**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "cra-template": "1.2.0",
+    "particles.js": "^2.0.0",
```
Comment (nippaR) [inline client/package.json:7]: unnecessary dependency i forgot remove it

File: client/src/App.js [hunk 2]
```diff
@@ -3,15 +3,20 @@ import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import Homepage from "./components/Homepage/Homepage";
 import AboutPage from "./components/AboutPage/AboutPage"; // Import the AboutPage component
 import Services from "./components/Services/Services";
+import Login from "./Login&Register/Login";
```
Comment (nippaR) [inline client/src/App.js]: set route Login page

File: client/src/App.js [hunk 3]
```diff
@@ -3,15 +3,20 @@ import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
...
+import Register from "./Login&Register/Register";
```
Comment (nippaR) [inline client/src/App.js]: set route Register page

File: client/src/Login&Register/Login.js
Comment (nippaR) [inline client/src/Login&Register/Login.js:1]: this is  a login page ui

File: client/src/Login&Register/Register.js
Comment (nippaR) [inline client/src/Login&Register/Register.js:1]: this is a register page ui

File: client/src/Login&Register/foods.jpg
Comment (nippaR) [inline client/src/Login&Register/foods.jpg:1]: assets

File: client/src/Login&Register/login.css
Comment (nippaR) [inline client/src/Login&Register/login.css:1]: this is a login page css file

File: client/src/Login&Register/register.css
Comment (nippaR) [inline client/src/Login&Register/register.css:1]: this is a register page ui

**Review by Dinusha4118 — APPROVED** [review]
good

**Pull request merged**